ecommerce/models.py

```diff
@@ -194,6 +228,10 @@ def choices(cls):
         decimal_places=5,
         max_digits=20,
     )
+    # Flag to determine if the order is in review status - if it is, then
+    # we need to not step on the basket that may or may not exist when it is
+    # accepted
+    is_review = models.BooleanField(default=False)
```

Comment:
This would be better as an `@property` derived from `state` rather than its own column so we don't have to worry about keeping those synchronized.